app/domain/operations/people/create_or_update.rb

```diff
@@ -82,8 +82,11 @@ def no_infomation_changed?(params:)
       end
 
       def find_existing_person(params)
-        person = Person.by_hbx_id(params[:hbx_id]).first
-        return person if person.present?
+        inbound_hbx_id = params[:hbx_id]
```

Comment:
Minor suggestion, but might want to rename this variable since the create_or_update operation is used widely across the system.  While we're fixing a bug in the inbound ATP process at the moment, this variable may look odd to other devs operating in a different context in the future.